deploy/clusterrole.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Do we need to put the manifests here since it should be in OCM rep and managed by cluster-manager?

Reply:
@skeeey comments?